src/tracing/ipc/service/consumer_ipc_service.cc

```diff
@@ -355,6 +355,7 @@ void ConsumerIPCService::CloneSession(
   if (req.has_clone_trigger_delay_ms()) {
     args.clone_trigger_delay_ms = req.clone_trigger_delay_ms();
   }
+  args.output_file_fd = ipc::Service::TakeReceivedFD();
```

Comment:
Add a comment saying that the client sends the file descriptor in all cases, but we use it only  if the sesssion is write_into_file, and close it otherwise.